packages/dappmanager/src/db/stakerConfig.ts

```diff
@@ -4,11 +4,23 @@ import {
   ConsensusClientGnosis,
   ExecutionClientGnosis,
   ExecutionClientMainnet,
-  ExecutionClientPrater
+  ExecutionClientPrater,
+  StakerItemData
 } from "../types";
-import { dbMain } from "./dbFactory";
+import { dbCache, dbMain } from "./dbFactory";
 import { interceptGlobalEnvOnSet } from "./interceptGlobalEnvOnSet";
 
+// Cache
+
+const STAKER_ITEM_METADATA = "staker-item-metadata";
+
+export const stakerItemMetadata = dbCache.indexedByKey<StakerItemData, string>({
+  rootKey: STAKER_ITEM_METADATA,
```

Comment:
Why don't you just put "staker-item-metadata" here? It seems you don't use the constant in any other place.

Reply:
Checkout the rest of db files, they use that standard. So in order to not break the standard I keep it like that